examples/cloudbuild_enabled/main.tf

```diff
@@ -14,38 +14,40 @@
  * limitations under the License.
  */
 
-
-provider "google" {
-  version = "~> 3.5.0"
-}
-
-provider "google-beta" {
-  version = "~> 3.5.0"
-}
-
-provider "null" {
-  version = "~> 2.1"
-}
-
-provider "random" {
-  version = "~> 2.2"
-}
-
 /*************************************************
   Bootstrap GCP Organization.
 *************************************************/
 
 module "seed_bootstrap" {
-  source                  = "../.."
-  org_id                  = var.org_id
-  billing_account         = var.billing_account
-  group_org_admins        = var.group_org_admins
-  group_billing_admins    = var.group_billing_admins
-  default_region          = var.default_region
-  org_project_creators    = var.org_project_creators
-  sa_enable_impersonation = true
+  source               = "../.."
+  org_id               = var.org_id
+  billing_account      = var.billing_account
+  group_org_admins     = var.group_org_admins
+  group_billing_admins = var.group_billing_admins
+  org_project_creators = var.org_project_creators
+
+  project = {
+    project_id    = var.project_id
+    labels        = {}
```

Comment:
As mentioned in other comments, I think we should have default values to prevent the basic use case from being so verbose. Ideally we don't increase the number of inputs required, but rather give people the flexibility to customize things more easily.